README.md

```diff
@@ -1 +1,112 @@
-# characters-counter
+# Characters Counter Application 
+
+## Description
+This Python application contains a function count_characters_occurrence(), that returns the number of characters in the string occuring only
+    once.
+It uses caching for performance optimization, meaning that if the same string is passed to the function multiple times, the result is fetched from memory instead of recalculating.
+
+## Features 
+- Caching with @cache Decorator that optimizes repeated calls with the same input.
+- Uses Counter to quickly count character occurrences.
+- The function uses a filter combined with a lambda function to select only those characters that occur exactly once.
+- Has adjustable parameter for occurrence requirement.
+
+## Requirements
+- collections.Counter
+- functools.cache
+- pytest
+
+## Solution 
```

Comment:
I think we can omit solution from the readme.
From the next task, that will get bigger and no need to duplicate it here. Imagine you need to do a change and now you have to do it twice

Reply:
Ok, I'm just not confident about what exactly to write in file README.md, thanks for tips!